tests/functional/TestSmokeTest/tests/timeout_test.go

```diff
@@ -0,0 +1,126 @@
+// +build functional
+
+//
+// Copyright (c) 2015 The heketi Authors
+//
+// This file is licensed to you under your choice of the GNU Lesser
+// General Public License, version 3 or any later version (LGPLv3 or
+// later), or the GNU General Public License, version 2 (GPLv2), in all
+// cases as published by the Free Software Foundation.
+//
+package functional
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/heketi/heketi/pkg/glusterfs/api"
+	"github.com/heketi/heketi/pkg/remoteexec/ssh"
+	"github.com/heketi/heketi/pkg/testutils"
+	"github.com/heketi/tests"
+)
+
+var (
+	nodes         = 4
+	SSHPort       = "22"
+	preScriptFile = "/var/lib/glusterd/hooks/1/start/pre/S99timeout.sh"
+)
+
+func getNodes() []string {
+	nodelist := make([]string, nodes)
+
+	for index, ip := 0, 100; index < nodes; index, ip = index+1, ip+1 {
+		nodelist[index] = "192.168.10." + fmt.Sprintf("%v", ip)
+	}
+
+	return nodelist
+}
```

Comment:
Don't do this. We have better ways of determining the gluster node IPs under test. Please refer to the cluster mgmt types.

Reply:
this was the function copied from https://github.com/heketi/heketi/blob/master/tests/functional/TestManyBricksVolume/tests/heketi_test.go#L53

Reply:
Ick. Well, now I have a task to clean up that ugly stuff. :-)
Let's not copy something that suboptimal.

Reply:
>Don't do this. We have better ways of determining the gluster node IPs under test. Please refer to the cluster mgmt types.

@phlogistonjohn am not able to find cluster mgmt types. can you point me there?

Reply:
I was specifically thinking about https://github.com/heketi/heketi/blob/master/pkg/testutils/cluster.go#L30 and https://github.com/heketi/heketi/blob/master/pkg/testutils/cluster.go#L59